gma/src/common/native_ad_internal.cc

```diff
@@ -59,6 +61,20 @@ Future<AdResult> NativeAdInternal::GetLoadAdLastResult() {
       future_data_.future_impl.LastResult(kNativeAdFnLoadAd));
 }
 
+void NativeAdInternal::insert_image(const NativeAdImage& image,
+                                    std::string image_type) {
+  if (image_type == std::string("icon")) {
+    icon_ = image;
+  } else {
+    images_.push_back(image);
+  }
+}
+void NativeAdInternal::clear_existing_images() {
+  if (images_.size() > 0) {
```

Comment:
It should be ok to call clear() directly. https://en.cppreference.com/w/cpp/container/vector/clear

Reply:
Done.